Dockerfile

```diff
@@ -95,13 +93,21 @@ RUN if test -e modules/reana-commons; then \
       fi \
     fi
 
+# hadolint ignore=DL3013
+RUN if [ "${DEBUG}" -gt 0 ]; then pip install --no-cache-dir -e ".[debug,xrootd]"; else pip install --no-cache-dir ".[xrootd]"; fi;
+
 # Check for any broken Python dependencies
 RUN pip check
 
 # Set useful environment variables
 ENV TERM=xterm \
     PYTHONPATH=/workdir
 
+# Create and switch to REANA user to be able to create snakemake-specific
+# directories in the home folder.
+RUN useradd reana --uid 1000 --create-home
+USER reana
+
```

Comment:
After discussing with Tibor, given that the UID of the user can be changed with `WORKFLOW_RUNTIME_USER_UID`, maybe it's better if we directly create the directory in the Dockerfile with `mkdir /.cache/snakemake` and then we set the env variable `XDG_CACHE_HOME` to `/.cache`? In this way, independently by which user is used, the directory always exists